tests/annotations/tests.py

```diff
@@ -148,6 +148,19 @@ def test_mixed_type_annotation_numbers(self):
         combined = int(test.pages + test.rating)
         self.assertEqual(b.combined, combined)
 
+    def test_empty_expression_annotation(self):
+        books = Book.objects.annotate(
+            selected=ExpressionWrapper(Q(pk__in=[]), output_field=BooleanField())
```

Comment:
@jarshwah -- is this test okay to commit despite your comment on the ticket, "ExpressionWrapper was never meant to be used to wrap up Q() objects. Q() are not really expressions themselves, they just implement some of the same API internally."?